frontends/main/src/app-pages/ErrorPage/ErrorPageTemplate.tsx

```diff
@@ -1,30 +1,40 @@
 import React from "react"
-import {
-  Container,
-  MuiCard,
-  CardContent,
-  CardActions,
-  ButtonLink,
-} from "ol-components"
+import { Container, ButtonLink, Typography, styled } from "ol-components"
 import { HOME } from "@/common/urls"
 
 type ErrorPageTemplateProps = {
   title: string
   children: React.ReactNode
 }
 
-const ErrorPageTemplate: React.FC<ErrorPageTemplateProps> = ({ children }) => {
+const ErrorContainer = styled(Container)(({ theme }) => ({
+  backgroundColor: theme.custom.colors.white,
+  borderRadius: "8px",
+  marginTop: "4rem",
+  padding: "16px",
+  boxShadow:
+    "1px 2px 1px -1px rgba(0,0,0,0.2),0px 1px 1px 0px rgba(0,0,0,0.14),0px 1px 3px 0px rgba(0,0,0,0.12)",
+}))
```

Comment:
The box shadow there is a style from MuiCard.

Rather than importing MuiCard in `main`, I just copied the boxshadow to preserve the existing behavior.